web/src/ui/timeline/content/PolicyRuleBody.tsx

```diff
@@ -0,0 +1,28 @@
+import { PolicyRuleContent } from "@/api/types"
+import EventContentProps from "./props.ts"
+
+const BanPolicyBody = ({ event, sender }: EventContentProps) => {
+	const content = event.content as PolicyRuleContent
+	const prevContent = event.unsigned.prev_content as PolicyRuleContent | undefined
+
+	if (prevContent !== undefined) {
+		// all fields for content are missing (this is against spec?) so we need to use the prev event's content
```

Comment:
Removing all fields is the correct way remove a policy

Reply:
Are you sure? The spec says for m.policy.rule.* that all three fields are *required*, and I don't see anything saying they should be removed for unbans

Reply:
Yes, but looks like the words were lost when writing the spec. MSC2313 says

> Invalid or missing fields are to be treated as though the rule doesn't exist. This is to allow for rules to be deleted while state events cannot be deleted in Matrix.

You can open a spec PR to fix that

Reply:
Alright, I'll go with it then. I'll consider opening an MSC after.

Reply:
It doesn't need a new MSC since the original already says that, just a spec PR is enough